net-wireless/uhd/uhd-4.0.0.0.ebuild

```diff
@@ -22,33 +21,41 @@ KEYWORDS="~amd64 ~arm ~x86"
 IUSE="+b100 +b200 doc e300 examples +mpmd octoclock +n230 test +usb +usrp1 +usrp2 +utils +x300"
 RESTRICT="!test? ( test )"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}
-			b100? ( usb )
-			b200? ( usb )
-			usrp1? ( usb )
-			usrp2? ( usb )
-			|| ( b100 b200 e300 mpmd n230 usrp1 usrp2 x300 )"
+	b100? ( usb )
+	b200? ( usb )
+	usrp1? ( usb )
+	usrp2? ( usb )
+	|| ( b100 b200 e300 mpmd n230 usrp1 usrp2 x300 )
+"
 
 RDEPEND="${PYTHON_DEPS}
-	e300? ( virtual/udev )
-	usb? ( virtual/libusb:1 )
 	dev-libs/boost:=
+	e300? ( virtual/udev )
 	sys-libs/ncurses:0[tinfo]
+	usb? ( virtual/libusb:1 )
 	$(python_gen_cond_dep '
 	dev-python/numpy[${PYTHON_MULTI_USEDEP}]
 	dev-python/requests[${PYTHON_MULTI_USEDEP}]
 	')
 "
 
-DEPEND="${RDEPEND}
+DEPEND="${RDEPEND}"
+
+BDEPEND="
+	app-arch/gzip
+	app-arch/unzip
 	doc? ( app-doc/doxygen )
 	$(python_gen_cond_dep '
 	dev-python/mako[${PYTHON_MULTI_USEDEP}]
 	')
-	app-arch/unzip
-	app-arch/gzip
 "
 
-PATCHES=( "${FILESDIR}/${PN}-4.0.0.0-tinfo.patch" )
+PATCHES=(
+	"${FILESDIR}/${P}-tinfo.patch"
+	"${FILESDIR}/${P}-fix-frequency-offsets.patch"
+	"${FILESDIR}/${P}-fix-gcc11-sleep-for.patch"
+	"${FILESDIR}/${P}-fix-boost-math-header.patch"
```

Comment:
the patches themself look good, but they're too involved for not doing a revbump, we need some actually mileage on these.